Dockerfile

```diff
@@ -65,3 +43,12 @@ RUN if [ "${DB}" = "oracle" ]; then unzip /opt/oracle/instantclient-basiclite-li
  && (cd /opt/oracle/instantclient_12_2/ && ln -s libclntsh.so.12.1 libclntsh.so) \
  && rm -rf /opt/system/vendor/oracle \
  && rm -rf /opt/oracle/*.zip; fi
+
+USER root
+RUN mkdir -p  /opt/system/tmp/sessions/ \
```

Comment:
`rake tmp:create` should do the job right?

Reply:
@hallelujah the problem is that the gems are not installed for root and we would need to install it again. So it fails.

Reply:
awesome!